providers/dnsimple/dnsimpleProvider.go

```diff
@@ -279,7 +280,7 @@ func (c *dnsimpleProvider) getClient() *dnsimpleapi.Client {
 
 	// new client
 	client := dnsimpleapi.NewClient(tc)
-	client.SetUserAgent("DNSControl")
+	client.SetUserAgent(fmt.Sprintf("%s/%s", "dnscontrol", version.Banner()))
```

Comment:
@tlimoncelli, @weppos better early than late, this change will conflict/break with https://github.com/StackExchange/dnscontrol/pull/2737.

Reply:
Thanks for the heads up @cafferata, really appreciated.

I don't want to cause any delay in this or the other PR. Therefore, I reverted the change. It was not essential and could be easily applied later on.